swagger.yaml

```
@@ -209,12 +208,11 @@ paths:
   /filters:
     post:
       tags:
-        - Public
-        - Private
+        - "Public"
       summary: Create a filter for a dataset
       description: >-
         Create a resource for listing a selection of dimensions and dimension
-        options to be added to filter for a dataset
+        options to be added to filter for a dataset.  Note - the is_area_type property is required to ensure a successful filter journey.
```

Just notices an extra whitespace here too
```suggestion
        options to be added to filter for a dataset. Note - the is_area_type property is required to ensure a successful filter journey.
```

---

Space removed.